src/main/java/seedu/financialplanner/storage/LoadData.java

```diff
@@ -95,37 +95,50 @@ private static boolean createNewFile() {
 
         return line.equalsIgnoreCase("y");
     }
-
+    private static boolean hasDescription(String[] split) {
+        return (split.length > 4);
+    }
     private static Cashflow getEntry(String type, String[] split)
             throws FinancialPlannerException, IllegalArgumentException {
         double value;
         int recur;
         Cashflow entry;
+        String description;
 
         switch (type) {
         case "I":
             value = Double.parseDouble(split[1].trim());
             recur = Integer.parseInt(split[3].trim());
+            if (hasDescription(split)) {
+                description = split[4].trim();
+            } else {
+                description = null;
+            }
```

Comment:
I think can extract the if-else block in both cases "I" and "E" into the hasDescription method and just make it getDescription?